slash/react/src/Form/Text/__tests__/TextInput.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import { TextInput } from "../TextInput";
+
+describe("TextInput", () => {
+  it("should have value", () => {
+    // Act
+    render(<TextInput label="TextInput label" value="Hello World" />);
+
+    // Assert
+    expect(screen.getByLabelText("TextInput label")).toHaveValue("Hello World");
+  });
+
+  it("should have default class", () => {
+    // Act
+    render(<TextInput label="TextInput label" value="Hello World" />);
+
+    // Assert
+    expect(screen.getByLabelText("TextInput label")).toHaveClass(
+      "af-form__input-text",
+    );
+  });
+
+  it("should have label text", () => {
+    // Act
+    render(<TextInput label="TextInput label" value="Hello World" />);
+
+    // Assert
+    expect(screen.getByText("TextInput label")).toHaveClass(
+      "af-form__group-label",
+    );
+  });
```

Comment:
Comme pour l'autre test je pense que ça peut-être réuni afin d'optimiser la durée des tests vu que le render est le même.

Reply:
done and resolve merci de ton aide 💓